code/src/main/java/com/onlinecontacttracing/storage/data-storage/Place.java

```diff
@@ -10,7 +10,7 @@
 @Index
 public class Place {
 
-  @Id String placeId;
```

Comment:
I take it place isn't a top-level entity, right? if it was we'd need an @Id.

Reply:
It is not an entity